### ISSUE TITLE
Tilesets with a margin not supported?

### ISSUE THREAD

**Issue by cawoodm** [craftyjs/Crafty#416]
It seems `Crafty.sprite()` doesn't support tilesets with a margin around the outside
![tmw_desert_spacing](https://f.cloud.github.com/assets/498834/141935/4135cfb0-7299-11e2-981a-9176b10515b9.png)

I had to add the paddingX to the x value on line 4475:

``` javascript
            x = temp[0] * (tile + paddingX)+paddingX;
            y = temp[1] * (tileh + paddingY)+paddingY;
```

**Comment by darrentorpey** [comment]
@cawoodm, FYI I'm taking this on for addition to the engine.

Out of curiosity, where did you get that nice tileset? Is it free to be used elsewhere outside of your game?

**Comment by cawoodm** [comment]
Super. That map came with the free "Tiled Map Editor" SW you can download at http://mapeditor.org It's cool because you can export your creation as JSON and then use the TiledMapBuilder (https://github.com/Kibo/TiledMapBuilder) component to employ the sprites.

**Comment by mucaho** [comment]
@starwed can you close this now, please? as it was actually the primary objective of #665 :)

**Comment by starwed** [comment]
Hah, sure.  :)

There is some magic phrase that will close an issue automatically when a PR lands -- I think if you include the words `Fixes #416` in the commit message it will happen?  

Although maybe you need the correct permissions.  No idea!  :)

**Comment by kevinsimper** [comment]
https://help.github.com/articles/closing-issues-via-commit-messages :+1:
